app/forms/idv/finance_form.rb

```diff
@@ -5,7 +5,10 @@ class FinanceForm
     FINANCE_TYPES = [:ccn, :mortgage, :home_equity_line, :auto_loan].freeze
 
     FINANCE_HTML_OPTIONS = {
-      ccn: { maxlength: 8 }
+      ccn: { class: 'ccn', maxlength: 8 },
```

Comment:
This other PR https://github.com/18F/identity-idp/pull/870 also touches this hash....so there may be some merge conflicts soon